components/DateForm.js

```diff
@@ -26,6 +26,13 @@ const Label = styled.label`
   font-weight: 400;
 `;
 
+const Select = styled.select`
+  border: 2px solid darkgreen;
+  border-radius: 0.5rem;
+  padding: 0.5rem;
+  background-color: white;
+`;
+
 export default function DateForm({ locationId, dates, setDates }) {
```

Comment:
You don't seem to be using `dates` in this component. At least not that i could see. If you are not using it, we should probably remove it.